### ISSUE TITLE
type for apiDoc missing in typescript definitions

### ISSUE THREAD

**Issue by F7502** [davesag/swagger-routes-express#95]
The recently added typescript definitions don't define a type for `apiDoc`:

https://github.com/davesag/swagger-routes-express/blob/6a5639377a767fed4345e01bef6348af0e3c17f0/index.d.ts#L38

When you have the typescript compiler option `noImplicitAny` activated then the build fails with:

```
11:11:11 INFO - node_modules/swagger-routes-express/index.d.ts(38,29): error TS7006: Parameter 'apiDoc' implicitly has an 'any' type.
```

**Comment by davesag** [comment]
Fixed in release `3.2.1` released just now.

Thanks for your report.

**Comment by F7502** [comment]
That was fast, thanks!
